### PR TITLE
make PercentLiteralDelimiters enforce %I() too

### PR DESCRIPTION
**Pull request by bronson**
It seems only natural to watch over %I() usage too.

-----------------

Before submitting the PR make sure the following are checked:

* [x] Wrote [good commit messages][1].
* [x] Commit message starts with `[Fix #issue-number]` (if the related issue exists).
* [x] Used the same coding conventions as the rest of the project.
* [x] Feature branch is up-to-date with `master` (if not - rebase it).
* [x] Squashed related commits together.
* [x] Added tests.
* [x] Added an entry to the [Changelog](../blob/master/CHANGELOG.md) if the new code introduces user-observable changes. See [changelog entry format](../blob/master/CONTRIBUTING.md#changelog-entry-format).
* [x] All tests are passing.
* [x] The new code doesn't generate RuboCop offenses.
* [x] The PR relates to *only* one subject with a clear title
  and description in grammatically correct, complete sentences.

[1]: http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html

### CODE REVIEW

File: spec/rubocop/cop/style/percent_literal_delimiters_spec.rb [hunk 1]
```diff
@@ -179,6 +180,28 @@
     end
   end
 
+  context '`%I` interpolated string array' do
```
Comment (bbatsov) [inline spec/rubocop/cop/style/percent_literal_delimiters_spec.rb]: string -> symbol :-)

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,7 @@
 ### New features
 
 * [#3615](https://github.com/bbatsov/rubocop/pull/3615): Add autocorrection for `Lint/EmptyInterpolation`. ([@pocke][])
+* Make PercentLiteralDelimiters enforce delimiters around %I() too. ([@bronson][])
```
Comment (bbatsov) [inline CHANGELOG.md]: Wrap the name of the cop and %I() in backticks.

**Comment by bbatsov** [comment]
Good catch! I've added a couple of small remarks. 

P.S. Please, capitalize the first word in commit messages.

**Comment by bronson** [comment]
Great suggestions, fixes coming.

Curious, are you interested in a PR removing 6 lines of context `ruby: 2` from the codebase (since Ruby 2.0 is now the minimum)?  For example: https://github.com/bbatsov/rubocop/blob/e63f764d7b44dc54fcf0727c2d340892c5d4f7ee/spec/rubocop/cop/style/percent_literal_delimiters_spec.rb#L169

**Comment by bbatsov** [comment]
> Curious, are you interested in a PR removing 6 lines of context ruby: 2 from the codebase (since Ruby 2.0 is now the minimum)?

Sure. We simply forgot about those.

**Comment by bronson** [comment]
remarks addressed.

**Comment by bronson** [comment]
and https://github.com/bbatsov/rubocop/pull/3617.  They only stood out because this is such a cruft-free codebase.  :)

**Pull request merged**